bomsaway.py

```diff
@@ -546,21 +546,21 @@ def on_export(self, event):
         with open(base+ext, 'w') as csvfile:
             wrt = csv.writer(csvfile)
 
-            wrt.writerow(['Refs', 'Value', 'Footprint',
-                          'QTY', 'MFR', 'MPN', 'SPR', 'SPN'])
+            wrt.writerow(['Quantity','Value', 'Refs', 'Footprint',
+                           'Manufacturer','Manufacture Part Number',
+                           'LCSC Part Number'])
```

Comment:
Sorry, but I'd rather not hardcode the name of a supplier into the code (SPN == Supplier Part Number).

Reply:
If LCSC can handle a more generic field name, the rest of this looks fine.

Reply:
@darkomen : can you give https://github.com/Jeff-Ciesielski/Boms-Away/commit/8c9f02d8d30a336ca8d494bf3a8f24f70fb5e942 a try and see if the LCSC.com exporter works?

Reply:
Hi Jeff! yes you commit works correctly! You're right, it's better in this way. Certainly I doesn't try with this format :)